pkgs/applications/networking/3proxy/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, coreutils }:
+
+stdenv.mkDerivation rec {
+  pname = "3proxy";
+  version = "0.8.13";
+
+  src = fetchFromGitHub {
+    owner = "z3APA3A";
+    repo = "3proxy";
+    rev = version;
+    sha256 = "1k5rqldiyakhwhplazlhswkgy3psdkpxhn85605ncwaqx49qy8vk";
+  };
+
+  makeFlags = [
+    "INSTALL=${coreutils}/bin/install"
+    "DESTDIR=$(out)"
```

Comment:
Do we need this? Please see  #65718.

Reply:
even if it is necessary should `placeholder out` be used?

Reply:
Replaced with ```"prefix=$(out)"```, didn't work without.